### PR TITLE
Upgrade GitHub Actions

### PR DESCRIPTION
**Pull request by cclauss**
Thanks for contributing!

- [ ] Have you updated CHANGELOG.md?

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 14
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v23
```
Comment (zbjornson) [inline .github/workflows/ci.yaml:83]: Oops, typo.

Comment (cclauss) [reply .github/workflows/ci.yaml]: Thx!

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -38,14 +38,14 @@ jobs:
...
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Install Dependencies
         run: |
           Invoke-WebRequest "https://ftp-osl.osuosl.org/pub/gnome/binaries/win64/gtk+/2.22/gtk+-bundle_2.22.1-20101229_win64.zip" -OutFile "gtk.zip"
           Expand-Archive gtk.zip -DestinationPath "C:\GTK"
           Invoke-WebRequest "https://downloads.sourceforge.net/project/libjpeg-turbo/2.0.4/libjpeg-turbo-2.0.4-vc64.exe" -OutFile "libjpeg.exe" -UserAgent NativeHost
           .\libjpeg.exe /S
-          npm install -g node-gyp@8
+          npm install -g node-gyp@9
```
[the remark this answers is not in the document]
Comment (cclauss) [reply .github/workflows/ci.yaml]: OK.  Back to node-gyp 8... https://github.com/nodejs/release#readme

**Review by zbjornson — APPROVED** [review]
Thanks!

**Pull request merged**